docker/backend-dev/conf/Config2.pm

```diff
@@ -61,18 +76,29 @@ $mongodb = "off";
 $mongodb_host = "mongodb://mongodb:27017";
 $mongodb_timeout_ms = 50000; # config option max_time_ms/maxTimeMS
 
-$memd_servers = [ "memcached:11211" ];
+$memd_servers = [ "127.0.0.1:11211" ];
```

Comment:
Not correct for all dev containers (local development)

Reply:
It's fine to keep the existing `memcached:11211` here, since the containers are running on the same VM.